edocumentation-domain/src/main/java/org/dcsa/edocumentation/domain/persistence/entity/ConsignmentItem.java

```diff
@@ -31,6 +31,9 @@ public class ConsignmentItem {
   @Column(name = "description_of_goods", nullable = false)
   private String descriptionOfGoods;
 
+  @Column(name = "hs_code", nullable = false)
+  private String hsCode;
```

Comment:
I know I am on vacation - but this should be a list in the API ;-)

Reply:
If you are on vacation, then don't review PRs. :)

Also, I know - this being a list is one of the things left for a future PR.